nupic/frameworks/opf/clamodel.py

```diff
@@ -419,7 +423,7 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
```

Comment:
Since this isn't used internally can I suggest that clients calling the `run` method that want to know how long each run takes simply time it themselves? Or did you have some use case in mind that that wouldn't work for?